apps/web/components/ui/PoweredByCal.tsx

```diff
@@ -17,15 +17,15 @@ const PoweredByCal = () => {
             <img
               className="relative -mt-px inline h-[10px] w-auto dark:hidden"
               src="/cal-logo-word.svg"
-              alt="Cal.com Logo"
+              alt={`${WEBSITE_DOMAIN} Logo`}
             />
           }
           {
             // eslint-disable-next-line @next/next/no-img-element
             <img
               className="relativ -mt-px hidden h-[10px] w-auto dark:inline"
               src="/cal-logo-word-dark.svg"
-              alt="Cal.com Logo"
+              alt={`${WEBSITE_DOMAIN} Logo`}
             />
```

Comment:
this should always stay cal.com

Reply:
@PeerRich with regard to whiltelabel, this make no sense...

Reply:
Unless we provide some way to get this logo dynamically it will stay as a Cal.com logo and would need code changes to modify anyways